src/infra/core/Cargo.toml

```diff
@@ -69,23 +80,23 @@ aws-sdk-s3 = { version = "0.35" }
 aws-smithy-http = { version = "0.57", features = ["rt-tokio"] }
 aws-smithy-types = { version = "0.57" }
 aws-credential-types = { version = "0.57" }
-trust-dns-resolver = "0.23"  # TODO: Needed for DNSLink resolution with IPFS
+trust-dns-resolver = "0.23"                                     # TODO: Needed for DNSLink resolution with IPFS
```

Comment:
minor: is this formatting necessary?

Reply:
For some reason my `better-toml` auto-formatter thinks it is... reverted